minigrid/wrappers.py

```diff
@@ -323,7 +323,7 @@ def __init__(self, env, tile_size=8):
         )
 
     def observation(self, obs):
-        rgb_img = self.get_frame(highlight=True, tile_size=self.tile_size)
+        rgb_img = self.get_frame(highlight=self.env.highlight, tile_size=self.tile_size)
```

Comment:
If wrappers can't modify this value, could you use `self.unwrapped.highlight` as if `self.env` is not the base environment, this will break in Gymnasium v1.0.0

Reply:
Done. 

I also changed `self.env.height` and `self.env.width` to use self.unwrapped, a few lines up.